docker-compose-watcher.yml

```diff
@@ -7,20 +7,19 @@ services:
       context: ./
```

Comment:
build directives need to be removed as this is not meant for development but for the community.